orchestration-shared-test/src/main/java/uk/gov/di/orchestration/sharedtest/basetest/IntegrationTest.java

```diff
@@ -341,5 +341,10 @@ public Optional<String> getIPVCapacity() {
         public boolean isDestroyOrchSessionOnSignOutEnabled() {
             return true;
         }
+
+        @Override
+        public boolean isReturnAuthTimeInIdTokenEnabled() {
+            return false;
```

Comment:
Note I've deliberately set this `false` here and in unit tests to match production. When the feature flag is enabled in prod the tests will need to be updated.